tests/test_checksum.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2022 spdx contributors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pytest
+
+from spdx.checksum import ChecksumAlgorithm
+
+
+@pytest.mark.parametrize("algorithm,expected",
+                         [("SHA1", "checksumAlgorithm_sha1"), ("SHA224", "checksumAlgorithm_sha224"),
+                          ("SHA3_256", "checksumAlgorithm_sha3_256"), ("BLAKE2B_256", "checksumAlgorithm_blake2b256"),
+                          ("MD5", "checksumAlgorithm_md5")])
+def test_checksum_to_rdf(algorithm, expected):
+    test_algorithm = ChecksumAlgorithm[algorithm]
+    rdf_algorithm = test_algorithm.checksum_to_rdf()
+
+    assert rdf_algorithm == expected
+
+
+@pytest.mark.parametrize("expected,rdf_algorithm",
+                         [("SHA1", "checksumAlgorithm_sha1"), ("SHA224", "checksumAlgorithm_sha224"),
+                          ("SHA3_256", "checksumAlgorithm_sha3_256"), ("BLAKE2B_256", "checksumAlgorithm_blake2b256"),
+                          ("MD5", "checksumAlgorithm_md5")])
+def test_checksum_from_rdf(rdf_algorithm, expected):
+    algorithm = ChecksumAlgorithm.checksum_from_rdf(rdf_algorithm)
+
+    assert algorithm == expected
+
+
+@pytest.mark.parametrize("expected,rdf_algorithm",
+                         [("SHA1", "_checksumAlgorithm_sha1"), ("SHA224", "checksumAlgorithm_sha_224"),
+                          ("SHA3_256", "checksumAlgorithm_sha3256"), ("BLAKE2B_256", "checksumAlgorithm_blake2b 256"),
+                          ("BLAKE2B_256", "checksumAlgorithm_blake2b-256"),
+                          ("BLAKE2B_256", "checksumAlgorithm_bblake2b 256")])
+def test_checksum_from_wrong_rdf(rdf_algorithm, expected):
+    algorithm = ChecksumAlgorithm.checksum_from_rdf(rdf_algorithm)
+
+    assert algorithm != expected
```

Comment:
I find this a very weird and vague test assertion. I still feel the `checksum_from_rdf` method should return a `Checksum` instance and should raise an Error if an invalid value was passed as an argument. The test should then check that a corresponding error is raised